db/schema.rb

```diff
@@ -173,9 +156,6 @@
     t.index ["code", "accredited_body_code"], name: "index_courses_on_code_and_accredited_body_code", unique: true
   end
 
-  create_table "data_migrations", primary_key: "version", id: :string, force: :cascade do |t|
```

Comment:
?

Reply:
This is what happens when you don't run data migration locally, there's no actual migration file for "data_migrations" table.  Best to remove this change from commit.